donation-api/mvnw

```diff
@@ -76,7 +76,7 @@ if [ -z "$M2_HOME" ] ; then
   ## resolve links - $0 may be a link to maven's home
   PRG="$0"
 
-  # need this for relative symlinks
+  # calloutNeed this for relative symlinks
```

Comment:
This shouldn't be changed as it's a maven file and not based on the project.

Reply:
Should I add this to a .gitignore? I obviously didn't manually change this as I didn't even know this file existed